<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrobbleSettingsWidget</class>
 <widget class="QWidget" name="ScrobbleSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="scrobblingOn">
     <property name="title">
      <string>Enable scrobbling</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Scrobble at</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="scrobblePoint">
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="tracking">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="percentText">
          <property name="text">
           <string notr="true">50</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>percent of the track</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="enfocreScrobbleTimeMax">
        <property name="text">
         <string>...or at 4 minutes (whichever comes first)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="podcasts">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Scrobble podcasts</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="allowFingerprint">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Allow Last.fm to fingerprint my tracks</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="directoriesBox">
     <property name="title">
      <string>Selected directories will not be scrobbled</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="CheckFileSystemView" name="exclusionDirs">
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckFileSystemView</class>
   <extends>QTreeView</extends>
   <header>../Settings/CheckFileSystemView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
